controllers/Controller.js

```diff
@@ -37,9 +37,9 @@ class Controller {
         }
     }
     async update(req, res, next) {
```

Comment:
add comments to clarify the purpose of this method and its parameters

Reply:
Why this method and not all the other methods?

Reply:
because this is the code that was in the PR. 
but you are right, it's relevant for all the other methods, and so on other comments are relevant in few functions.